src/ua/griddynamics/Main.java

```diff
@@ -0,0 +1,46 @@
+package ua.griddynamics;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("How many mines do you want on the field? >");
+        int numberOfMines = scanner.nextInt();
+
+        char[][] hiddenField = new char[Game.HEIGHT][Game.WIDTH];
+        char[][] visibleField = new char[Game.HEIGHT][Game.WIDTH];
```

Comment:
hiddenField and visibleField is internal business logic 
We as a user don't care about it